Sources/BraintreePayPal/BTPayPalClient.swift

```diff
@@ -75,6 +75,10 @@ import BraintreeDataCollector
         _ request: BTPayPalVaultRequest,
         completion: @escaping (BTPayPalAccountNonce?, Error?) -> Void
     ) {
+        if request.userAuthenticationEmail != nil {
+            webAuthenticationSession.prefersEphemeralWebBrowserSession = true
+        }
```

Comment:
@scannillo Curious if you also run into weird behavior with MPE email without this. Even with clearing cookies on safari I see a "transition failed" message in the webview. It seems like we can't have any cookies in the session for this feature.

Reply:
were you seeing this in sandbox?

Reply:
Production - when prod is fixed I'll remove this and see if the behavior is still unexpected. This flag removes all cookies from the `ASWebAuthenticationSession` but may not be necessary when prod is fixed.

Reply:
I honestly forgot this ephemeral session setting was changed for this feature. I'm not sure we want to do this anymore 🤔

Reply:
```suggestion
```